cmd/gitserver/server/serverutil.go

```diff
@@ -551,6 +555,17 @@ func mapToLog15Ctx(m map[string]any) []any {
 	return ctx
 }
 
+// mapToLogCtx translates a map to log context fields.
+func mapToLogCtx(m map[string]any) string {
```

Comment:
```suggestion
func mapToLoggerField(m map[string]any) log.Field {
```

This way, we can turn the map into a `log.Object` rather than turning it into a big string.  

Also, making it stable as the original function did is necessary.